spec-main/api-net-spec.ts

```diff
@@ -1075,20 +1075,6 @@ describe('net module', () => {
       await emittedOnce(urlRequest, 'abort');
     });
 
-    it('should not follow redirect when mode is error', async () => {
-      const serverUrl = await respondOnce.toSingleURL((request, response) => {
-        response.statusCode = 302;
-        response.setHeader('Location', '/200');
-        response.end();
-      });
-      const urlRequest = net.request({
-        url: serverUrl,
-        redirect: 'error'
-      });
-      urlRequest.end();
-      await emittedOnce(urlRequest, 'error');
-    });
-
```

Comment:
uh? why is this test being deleted?

Reply:
https://electronhq.slack.com/archives/C5VT8SQ8K/p1599188065017200

Reply:
I don't understand why the destroy event would prevent the behavior that this test is testing from changing. We should still not follow redirects when the mode is 'error'.

Reply:
fixed